about.html

```diff
@@ -0,0 +1,189 @@
+        </section>
+      </div>
+      <div class="row">
+        <section class="healthyRecipes">
+          <div>
+            <h2>See Easy and Cheap Healthy Recipes</h2>
+            <hr />
+            <p class="a">
+              Take a look at the easiest healty recipes which takes <br />
+              no time to prepare it .
+            </p>
+            <p class="b">
+              Take a look at the easiest healty recipes which takes
+              no time to prepare it .
+            </p>
+          </div>
+          <div class="container">
+          <div class="recipes1">
+            <div class="recipeImage">
+              <img
+                src="assets/images/Coconut-and-corn-curry-3ef570b.jpg"
+                alt="Coconuts"
+                class="sepia"
+              />
+            </div>
+            <div class="centerd">
+              <h3>Coconut & corn curry</h3>
+              <p>
+                Put the lentils in a sieve and rinse until the water runs clean.
+                Put the oil in a saucepan over a medium heat, tip in the onion
+                and cook for 8 mins until softened. Add the ginger, turmeric and
+                curry powder, and cook for another minute until fragrant. Stir
+                in the lentils. Tip in 400ml hot water and bring to the boil.
+                Turn the heat down to medium and simmer for 15 mins until most
+                of the water has been absorbed. Add the baby corn and 100ml
+                water, cover and continue to simmer for 5-10 mins until the
+                lentils are soft and the corn still has a little bite. Stir in
+                the coconut yogurt to warm up, but don’t let it boil. Serve
+                topped with more coconut yogurt, if you like, and coriander
+                leaves sprinkled over.
+              </p>
+            </div>
+          </div>
+          <div class="recipes1">
+            <div class="recipeImage">
+              <img
+                src="assets/images/Miso-and-butternut-soup-efe9277.jpg"
+                alt="Miso"
+                class="sepia"
+              />
+            </div>
+            <div class="centerd">
+              <h3>Miso & butternut soup</h3>
+              <p>
+                Heat the oil in a large pan and fry the onion for 5 mins to
+                soften. Add the squash and garlic, and stir for a minute. Add
+                the beans and bouillon along with a litre of water, then cover
+                and simmer for 20 mins until the squash is tender. Meanwhile,
+                steam the kale for 10 mins, then toss together with the sesame
+                oil, seeds and ginger. Add the miso to the soup, then blitz
+                until smooth using a hand blender. Pour into bowls and top with
+                the sesame kale mix to serve
+              </p>
+            </div>
+          </div>
+          </div>
+        </section>
+      </div>
```

Comment:
- [ ]  Good job for working on " About " page as suggested by the previous reviewer, still you have to address the some of  previous reviewer suggestion.
- [ ] You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:  Layout: composition and space between elements
- [ ] Kindly adjust Layout: composition and space between elements just like the design guidelines : For Mobile screen size the "healthy-Recipes" section are overlapping .
Your design Look likes this : 
![Cap12](https://user-images.githubusercontent.com/93598189/190154254-4a3fed9b-c639-4939-a92a-09081187ae7b.PNG)

The Template design look likes this : 
![te](https://user-images.githubusercontent.com/93598189/190154597-54b5c095-a72e-48b9-a427-0ba1a0d3520b.PNG)

Reply:
fixed